content/en/docs/test.md

```diff
@@ -369,7 +369,9 @@ Notes catch the reader's attention without a sense of urgency.
 
 You can have multiple paragraphs and block-level elements inside an admonition.
 
-| Or | a | table |
+| Or | Even |
+|---|-----|
+| A | Table |
```

Comment:
Would you like to align the table column?

BTW, I think lower case would make more sense.

Reply:
```suggestion
| or | even |
|:---:|:-----:|
| a | table |
```
Is that right?

Reply:
Sure, I updated it. 🙏